pkgs/by-name/rs/rsshub/0002-fix-network-call.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/scripts/workflow/build-routes.ts b/scripts/workflow/build-routes.ts
+index 9807cfc..b9dcfb9 100644
+--- a/scripts/workflow/build-routes.ts
++++ b/scripts/workflow/build-routes.ts
+@@ -4,6 +4,7 @@ import { parse } from 'tldts';
+ import fs from 'node:fs';
+ import path from 'node:path';
+ import toSource from 'tosource';
++import { exit } from 'node:process';
+
+ import { getCurrentPath } from '../../lib/utils/helpers';
+ const __dirname = getCurrentPath(import.meta.url);
+@@ -73,3 +74,5 @@ fs.writeFileSync(path.join(__dirname, '../../assets/build/radar-rules.js'), `(${
+ fs.writeFileSync(path.join(__dirname, '../../assets/build/maintainers.json'), JSON.stringify(maintainers, null, 2));
+ fs.writeFileSync(path.join(__dirname, '../../assets/build/routes.json'), JSON.stringify(namespaces, null, 2));
+ fs.writeFileSync(path.join(__dirname, '../../assets/build/routes.js'), `export default ${JSON.stringify(namespaces, null, 2)}`.replaceAll(/"module": "(.*)"\n/g, `"module": $1\n`));
++
++exit(0);
```

Comment:
Do the generated files (routes.json, maintainers.json, ...) look okay, compared to the previous release?

Reply:
Without the patch, you still get the error, right? It would be good to know if the error influences the output in some way. I'll run the script outside the sandbox to test.

Reply:
I built it outside the sandbox for the above diff, not via nix-build but running phases one by one using a helper script https://github.com/phanirithvij/system/blob/main/scripts/nix-develop-interactive.bash

Reply:
Alright it seems the error is not relevant.
```
total 9948
-rw-r--r-- 1 arne users  149708 2025-05-31 13:14 maintainers.json
-rw-r--r-- 1 arne users  802250 2025-05-31 13:14 radar-rules.js
-rw-r--r-- 1 arne users 1010221 2025-05-31 13:14 radar-rules.json
-rw-r--r-- 1 arne users 4104476 2025-05-31 13:14 routes.js
-rw-r--r-- 1 arne users 4110235 2025-05-31 13:14 routes.json
```